packages/expo-clipboard/src/Clipboard.ts

```diff
@@ -14,9 +15,13 @@ const onClipboardEventName = 'onClipboardChanged';
 
 type ClipboardEvent = {
   /**
-   * The new content of the user's clipboard.
+   * @deprecated Deprecated. Returns empty string. Use [`getStringAsync()`](#getstringasyncoptions) instead to retrieve clipboard content.
```

Comment:
Do we need additional "deprecated" here? 🤔 Isn't `@deprecated` enough?

Reply:
Yes, currently the docs don't display the "Deprecated" header implicitly. I think @Simek is already aware of this 😉

Reply:
Yup, it's something that I want to correct, it has be raised a few times already. :)

It's probably also a bit confusing now, because when there is no message after the deprecation annotation, the "Deperecated" word is added then.